plugins/nf-validation/src/main/nextflow/validation/SamplesheetConverter.groovy

```diff
@@ -116,11 +109,6 @@ class SamplesheetConverter {
                 }
             }
 
-            // Check for row uniqueness
-            if(!skipDuplicateCheck && this.rows.contains(row)) {
-                def Integer firstDuplicate = this.rows.findIndexOf { it == row }
-                this.errors << "The samplesheet contains duplicate rows for entry ${firstDuplicate + 1} and entry ${getCount()} (${row})".toString()
-            }
```

Comment:
Where has this functionality been moved to?

Reply:
draft `2020-12` has a keyword called `uniqueItems` that can be used on an array. This basically checks for duplicates, so no need to keep this in our code anymore. I've also tried to keep all validation in `validateParameters()` and not in `.fromSamplesheet()` anymore

Reply:
Hmm, I thought `uniqueItems` was for `type: array`, but our unqiueness check was for the overall array of the samplesheet (i.e. across rows). Maybe I need to see an example of how this works but I am not quite certain on this

Reply:
I've tested this and it does also work for an `array` of `properties`. Alternatively I added `uniqueEntries: array` which could also be used to check for duplicates

Reply:
Can you just point me to the test that covers this?